### ISSUE TITLE
importing ExportedImage gives below error

### ISSUE THREAD

**Issue by vittaljk** [Niels-IO/next-image-export-optimizer#14]
Unhandled Runtime Error
Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: object.

Check the render method of `ForwardRef(LoadableComponent)`.

**Comment by Niels-IO** [comment]
Hi @vittaljk,

Could you please share the code, how you implemented the ExportedImage component?

Best,
Niels

**Comment by primozs** [comment]
This happens to me if I upgrade from next ^12.1.4 to ^12.1.5

**Comment by Niels-IO** [comment]
Hey @primozs,

I just tested it with 12.1.5 and haven't gotten an error. Could you please share more info, also possible sample code?
Thanks 🙂

**Comment by Niels-IO** [comment]
Hi @primozs,

Thanks! I think it has to with the swc compilation to commonjs. I could reproduce the error, but have not found a way to fix it.

**Comment by primozs** [comment]
Hi, @Niels-IO I have custom build config I am not using swc and result is the same.

**Comment by Niels-IO** [comment]
Hey @primozs,

I tried a few things and I think the culprit is the dynamic import in the ExportedImage.tsx component:
```
import dynamic from "next/dynamic";

// Dynamic loading with SSR off is necessary as there might be a race condition otherwise,
// when the image loaded and errored before the JS error handler is attached
const Image = dynamic(() => import("next/image"), { ssr: false });
```

When I remove the dynamic import that is necessary to disable the ssr, the error goes away, but the race condition is there.

**Comment by primozs** [comment]
@Niels-IO Thank You for looking into it!
I am wondering why not use static dependency. Then user can still make dynamic import of ExportedImage? 

I have static dependency of ExportedImage;
Cheers

**Comment by Niels-IO** [comment]
Hi @primozs,

I think I was able to fix the issue in v0.9.6. Could you please try it in your end as well?

Thanks a lot!

**Comment by primozs** [comment]
@Niels-IO It works! Thank You!
